pyproject.toml

```diff
@@ -0,0 +1,20 @@
+[tool.poetry]
+name = "tanker-ruby-identity-sdk"
+version = "0.1.0"
+description = "Ruby Identity SDK for Tanker"
+authors = ["Tanker team <tech@tanker.io>"]
+
+[tool.poetry.dependencies]
+python = "^3.7"
+tankerci = { version = ">= 2020.12.3123", source = "gitlab" }
```

Comment:
Nitpick: maybe start by requiring a more recent version when introducing the requirement? E.g.  `2021.6.3413` which has been picked up initially.